test/e2e/common/container_probe.go

```diff
@@ -139,6 +139,7 @@ var _ = framework.KubeDescribe("Probing container", func() {
 		livenessProbe := &v1.Probe{
 			Handler:             execHandler([]string{"cat", "/tmp/health"}),
 			InitialDelaySeconds: 15,
+			TimeoutSeconds:      5, // default 1s can be pretty aggressive in CI environments with low resources
```

Comment:
This test started to flake once dockershim was updated to correctly return DeadlineExceeded grpc status error. It seems like the default 1s timeout was  too aggressive here, my guess is because `pull-kubernetes-e2e-node` is probably very resource starved. I updated this to 5 seconds and the test seems to be passing consistently. 

Increasing the timeout here seems safe, because this test is really expecting that the exit code from the probe returns 0, it should allow for the probe to take longer than the default 1s, as long as it doesn't return a non-zero exit code.

Reply:
should the same be on line 124?

Reply:
hmmm.. so that test wasn't failing because the test is expecting a restart, but now that you mention it may be failing due to timeout and not a non-zero exit code, so I think it may make sense to set a similar timeout there.